robotbase/src/main/java/com/team2357/frc2022/commands/DriveRunMotorCommand.java

```diff
@@ -0,0 +1,32 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.lib.commands.CommandLoggerBase;
+import com.team2357.lib.commands.DriveProportionalCommand;
+import com.team2357.lib.subsystems.drive.FalconTrajectoryDriveSubsystem;
+import com.team2357.lib.subsystems.drive.SkidSteerDriveSubsystem;
+
+public class DriveRunMotorCommand extends CommandLoggerBase { 
+    SkidSteerDriveSubsystem m_driveSub;
+    double m_speed;
+
+    public DriveRunMotorCommand(SkidSteerDriveSubsystem driveSub, double speed) {
```

Comment:
Should this have a turn variable as well? I'm not sure how useful a command that just moves forward or back will be.

Reply:
It's useful for burn ins, that is it. It can disappear.